src/transformers/modeling_utils.py

```diff
@@ -1490,6 +1491,11 @@ def load(module: nn.Module, prefix=""):
             start_prefix = cls.base_model_prefix + "."
         if hasattr(model, cls.base_model_prefix) and not has_prefix_module:
             model_to_load = getattr(model, cls.base_model_prefix)
+            if any(key in expected_keys_not_prefixed for key in loaded_keys) > 0:
```

Comment:
nit - why not just `if any(key in expected_keys_not_prefixed for key in loaded_keys):` ?

Reply:
Woopsy, had a `len` before that `any` and forgot to remove the comparison.